src/fides/api/ops/schemas/connection_configuration/connection_secrets_sovrn.py

```diff
@@ -22,7 +22,7 @@ class SovrnEmailSchema(ExtendedConsentEmailSchema):
     """
 
     third_party_vendor_name: str = "Sovrn"
-    recipient_email_address: str  # In production, use: privacy@sovrn.com
+    recipient_email_address: str = "privacy@sovrn.com"
```

Comment:
Thanks for adding this here - we had just been trying to be really careful in testing not to accidentally send this email to Sovrn. Since this is shown as the default even if the user has privately saved a separate email, it's easy to accidentally save this in testing.

But we do need to have this here for prod.

Reply:
Thanks for the background on `ConsentEmailSchema`. I originally created my own because I didn't see how the advanced settings would be relevant for an erasure email connector but when you mentioned wanting to save identity types directly on the connector it made sense. I renamed `ConsentEmailSchema` to `EmailSchema` and deleted the schema I created.